apps/src/redux/isRunning.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * This duck module tracks whether the current app is running or not.
+ */
+var TOGGLE_RUNNING = 'isRunning/TOGGLE_RUNNING';
+
+module.exports.default = function (state, action) {
+  var nextState = state || false;
+
+  if (action.type === TOGGLE_RUNNING) {
+    nextState = action.isRunning;
```

Comment:
Do you get the impression that we should write one reducer-creator for this sort of set-primitive thing?

```javascript
function buildSetPrimitiveReducer(actionKey, defaultValue) {
  return function (state, action) {
    state = state || defaultValue;
    if (action.type === actionKey) {
      state = action.value;
    }
    return state;
  };
}

function buildSetPrimitiveAction(actionKey) {
  return function (value) {
    return { type: actionKey, value: value };
  };
}

module.exports.default = buildSetPrimitiveReducer(TOGGLE_RUNNING, false);
module.exports.toggleRunning = buildSetPrimitiveAction(TOGGLE_RUNNING);
```